eng/jacoco-test-coverage/pom.xml

```diff
@@ -531,6 +531,22 @@
   </dependencies>
   <build>
     <plugins>
+      <plugin>
```

Comment:
Why is this needed now?

Reply:
This causes a Maven plugin to fail a build because that library isn't using a standard Track 2 groupId, in the parent POM we whitelist and `com.azure:*`, `com.azure.resourcemanager:*`, and `com.azure.spring:*` dependencies as we know those are track 2 and we allow any track 2 library to have a track 2 dependency. This artifact is using `com.microsoft` so it was causing build to fail.

cc: @srnagar

Reply:
We are still debating the namespace for this library. So, I think we can skip this library from coverage checks until the namespace is decided.